### ISSUE TITLE
Always remove the repo, do not ask

### ISSUE THREAD

**Issue by donquixote** [openeuropa/drupal-site-template#57]
The installer wizard asks a final question, whether to remove the git repo or not.

There are two problems with this:
- Choosing "n" (no) leads to a failed install.
- Choosing "n" (no) makes no sense, because you are not supposed to continue the git history.

**Comment by brummbar** [comment]
That's true, we will update the documentation to include the `--remove-vcs` [command option](https://getcomposer.org/doc/03-cli.md#create-project).
Thanks for the suggestion!
